sky/setup_files/setup.py

```diff
@@ -56,11 +56,8 @@ def parse_readme(readme: str) -> str:
 
 install_requires = [
     'wheel',
-    # NOTE: ray requires click>=7.0. Also, click 8.1.x makes our rendered CLI
-    # docs display weird blockquotes.
-    # TODO(zongheng): investigate how to make click 8.1.x display nicely and
```

Comment:
@concretevitamin Do you have an idea of which cli helps to test? It seems the helps look fine to me with the latest click.

Reply:
Maybe try both `sky * -h` and locally build the docs html?

Reply:
Seems `sky * -h` works fine.

For the docs HTML, it seems the problem before was due to the `sphinx-cli` version being too old. After upgrading the package, the problem went away.